third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -73,7 +74,7 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 								Schema: map[string]*schema.Schema{
 									"config": {
 										Type:     schema.TypeList,
-										Required: true,
+										Optional: true,
```

Comment:
Does it  make sense to add an `AtLeastOneOf` for config, versioned_expr, expr?

Reply:
Yes, I was going to actually put `ExactlyOneOf` here for `versioned_expr` or `expr` (because `config` is required with `versioned_expr` and `config.src_ip_ranges` conflicts with `expr`), but because `rule` is a list with no `MaxItems`, these won't work here.  So this is a good note to leave for when this does work.